src/pyfaf/queries.py

```diff
@@ -1287,6 +1288,17 @@ def get_crashed_unknown_package_nevr_for_report(db, report_id):
             .filter(st.ReportUnknownPackage.type == "CRASHED")
             .all())
 
+def get_crashed_package_version_for_report(db, report_id):
+    """
+    Return version for unknown packages that CRASHED in a given
+    report.
+    """
+    return (db.session.query(st.ReportUnknownPackage.version)
+            .filter(st.ReportUnknownPackage.report_id == report_id)
+            .filter(st.ReportUnknownPackage.type == "CRASHED")
+            .order_by(desc('version'))
```

Comment:
This ordering is not enough either - with packages there is system epoch:version-release and all of these needs to be compared in this order